### ISSUE TITLE
Measure Team Effectiveness 'report' lacks sufficient metrics to be actionable/valuable

### ISSUE THREAD

**Issue by sbohlen** [microsoft/vsts-extension-retrospectives#365]
First, kudos for providing the facility to capture more quantitative data about the health of the team/project with the above-referenced feature.  However, I'm frankly struggling to make actual *use* of the results to meaningfully inform our subsequent actions, and am hoping for some clarification around how this tool is expected to be leveraged.

Following is the "Retrospective Summary" output from a recent retrospective:
<img width="349" alt="screenshot-retro-summary" src="https://user-images.githubusercontent.com/186336/174064336-a04c093b-6c09-49b3-9fed-e686076a5389.png">

From this 'report', I can clearly see the average scores for each of the "Team Effectiveness" questions.  But since the tool is anonymous (a good choice, BTW), without at least a count of responses (either or both "number of forms completed" or "number of respondents per each question") I'm honestly challenged to make any meaningful informed conclusions from these values.

If these averages are the result of e.g., 5-10% of retro participants having completed the form, then the values aren't safely representative of anything but a statistically insignificant percentage of the team.  Whereas if they are the result of 70-100% of the retro participants completing the form, then they suddenly become actually meaningful.

In short, these metrics as presented back to us certainly are _data_, but IMO they aren't _actionable_ data in their present form.

A few Qs:

1. how are others actually making use of these scores, absent any transparency into response-rates?
2. is there some custom AzDO query I can perform e.g., the equivalent of `SELECT COUNT(DISTINCT(responder)) FROM TeamHealthResultsTable WHERE retrospectiveId=ABC` (and, yes, I know that's both bad T-SQL and that AzDO doesn't support T-SQL; that's just a conceptual example!)
3. Are there any plans to add add'l metrics e.g., number of respondents to the "Retrospective Summary" output?
4. Were we to undertake the work, would a PR from myself/my team to add this capability to the tool be considered for acceptance into the repo?

Thanks in advance!

**Comment by dstoddar** [comment]
@sbohlen , we recently released reporting capability. Please review and provide any feedback or comments! Thank you.

**Comment by sbohlen** [comment]
Thanks for the heads-up, @dstoddar  -- I just checked the report from one of our past retro team feedback data collection cycles and its waaaaay better now; nice work all around to you and the team!  Closing this issue accordingly.  Cheers!

**Comment by dstoddar** [comment]
@sbohlen , you are welcome. Unfortunately, there's a bug in the historical data (#383 ). The data is still there, just rendering incorrectly. :(. But all future retrospectives will be correct and we've prioritized resolution of this bug.
